docs: update docs for collect list/set (#7106)
confluentinc/ksql: docs/reference/server-configuration.md

ksql.fail.on.production.error=false
```

## `ksql.functions.collect_list.limit`

> Allows users to limit the size of the resultant Array to N entries, beyond which

Reviewer: ```suggestion
Limit the size of the resultant Array to N entries, beyond which
```

> any further values will be silently ignored, by setting this configuration to N.

Reviewer: ```suggestion
any further values are silently ignored, by setting this configuration to N.
```

Also see [aggregate-functions](/reference/aggregate-functions)

## `ksql.functions.collect_set.limit`

> Allows users to limit the size of the resultant Set to N entries, beyond which

Reviewer: ```suggestion
Limits the size of the resultant Set to N entries, beyond which
```

> any further values will be silently ignored, by setting this configuration to N.

Reviewer: ```suggestion
any further values are silently ignored, by setting this configuration to N.
```
